### ISSUE TITLE
CycloDS Evolution (World) (v1.1) (Unl) fails self test

### ISSUE THREAD

**Issue by Kurayami7** [Arisotura/melonDS#1096]
![Capture](https://user-images.githubusercontent.com/77198254/117063906-d19f1580-ad68-11eb-92bb-65ca38801ba9.PNG)

**Comment by Arisotura** [comment]
of course, we don't emulate flashcarts. it would be neat to do for the technical challenge and for preservation, but it's a low priority item -- melonDS can directly load homebrew, so there isn't much advantage to using this.

**Comment by Kurayami7** [comment]
Point taken. I won't bother reporting any more unlicensed stuff.
